spec/support/helpers.rb

```diff
@@ -318,7 +318,7 @@ def install_gems(*gems)
             Dir.chdir(root) { gem_command! :build, gemspec.to_s }
           end
           bundler_path = root + "bundler-#{Bundler::VERSION}.gem"
-        elsif g.to_s =~ %r{\A/.*\.gem\z}
+        elsif g.to_s =~ %r{\A[A-Z]?:/.*\.gem\z}
```

Comment:
Try

```suggestion
        elsif g.to_s =~ %r{\A(?:[A-Z]:)?/.*\.gem\z}
```

Reply:
Hm, only now I looked at this: What is the first `?:` for? Looking at it now, I think I should just have made the `?` apply to the `[A-Z]` and `:`.

Reply:
Yeah, `([A-Z]:)?` probably works too. The `?:` only indicates that the group does not need to be captured. See https://ruby-doc.org/core-2.6.1/Regexp.html#class-Regexp-label-Grouping.

Reply:
TIL 👍

Reply:
Which version do you prefer? I don't have a strong opinion.

Reply:
Yours of course, adds useful functionality. I didn't even know this was possible and sometimes avoided `(...)` because of that reason, so 👍

Reply:
Ok!